linux_install_scripts/gams.sh

```diff
@@ -7,6 +7,7 @@ ARG TARGET="linux_x64_64_sfx.exe"
 ARG GAMS_DIR="gams29.1_linux_x64_64_sfx"
 ENV GAMS_VERSION="29.1.0"
 ENV PATH="${PREFIX}/GAMS_${GAMS_VERSION}/${GAMS_DIR}:${PATH}"
+ENV LD_LIBRARY_PATH="${PREFIX}/GAMS_${GAMS_VERSION}/${GAMS_DIR}"
```

Comment:
For safety / future proofing, this needs to augment the LD_LIBRARY_PATH env var and not just overwrite it (like the PATH variable above).